.babelrc.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
```

Comment:
I'm not opposed to this change at all - just curious why it was necessary?

Reply:
for future things. I started exploring what would be involved in using jest to run the tests, and this PR is sort of "preparation" for that future. With jest, all the tests run inside node, so it can be helpful to have a babel config that changes depending on process.env so you have one config for node execution and a slightly different config for browser execution.

Reply:
Ah! Awesome to hear.